### ISSUE TITLE
Linux Issue with Publii

### ISSUE THREAD

**Issue by Areahints** [GetPublii/Publii#175]
I installed the latest .deb version of Publii from the downloads page on the website. After installation, the assigned shortcut created cannot launch the publii app, so I start publii as **root**. fires up well enough, ask me to create a website, which I do, I have already prepared a repo as I want to use Github pages with publii. I input all the information needed and click on the save button, but **there is no feedback** that it has saved successfully, however when I try testing the connection, it reports **Success! Application was able to connect with your server.** 

I assigned a folder for publii to use for preview and when I click on preview your changes, the app says its rendering assets and actually writes them to the folder, but I have to manually go and click the index.html in order to see the preview because the app doesn't do it.  the folder where publii previews appears as an icon with a lock because publii requires **root**

Please help!

```
uname -a
Linux areahints-HP-15-Notebook-PC 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by dziudek** [comment]
Your issue is most probably connected by fact that when you started Publii with root then all catalogs created by Publii also needs root permissions. I suppose that there must be something wrong with your linux configuration. Unfortunately it is hard to find a solution as every linux distribution is different (it is not like Windows or macOS). I have tested Publii on Ubuntu, Fedora and Linux Mint - in all these cases core Publii functions was working correctly - without root access.

**Comment by Areahints** [comment]
ok @dziudek I took your advice and uninstalled the deb package, I decided to use appimage instead because you can run it without root, so i downloaded, ran `chmod a+x`  and` ./appimage` but I got the following error

```
The window-config.json file will be created
The app-config.json file will be created
/tmp/.mount_Publii4u44Cy/resources/app/back-end/vendor/sql.js:5
1<process.argv.length?process.argv[1].replace(/\\/g,"/"):"unknown-program");f.arguments=process.argv.slice(2);"undefined"!==typeof module&&(module.exports=f);process.on("uncaughtException",function(a){if(!(a instanceof la))throw a;});f.inspect=function(){return"[Emscripten Module object]"}}else if(ga)f.print||(f.print=print),"undefined"!=typeof printErr&&(f.printErr=printErr),f.read="undefined"!=typeof read?read:function(){throw"no read() available (jsc?)";},f.readBinary=function(a){if("function"===
                                                                                                                                                                                                                               ^

Error: ENOENT: no such file or directory, open '/home/areahints/Documents/Publii/config/app-config.json'
    at Object.fs.openSync (fs.js:646:18)
    at Object.module.(anonymous function) [as openSync] (ELECTRON_ASAR.js:166:20)
    at Object.fs.writeFileSync (fs.js:1303:33)
    at App.loadConfig (/tmp/.mount_Publii4u44Cy/resources/app/back-end/app.js:388:16)
    at new App (/tmp/.mount_Publii4u44Cy/resources/app/back-end/app.js:62:14)
    at App.<anonymous> (/tmp/.mount_Publii4u44Cy/resources/app/main.js:96:19)
    at emitTwo (events.js:131:20)
    at App.emit (events.js:214:7)


```

**Comment by Areahints** [comment]
Hey! @dziudek  i was able to get it up and running by simply looking for every occurrence of publii that was created by the root install and deleting it. the app runs fine and I've learned something new about appimages! Thanks

**ps:** what libc library does publii depend on? I discovered this when trying to use gdebi to install the .deb package, apparently the lintian output reads that my system is missing a dependency on libc, if I knew which one it was, I can just go grab it from aptitude, this isn't really an issue but would be nice to know..

More Info here!
https://lintian.debian.org/tags/missing-dependency-on-libc.html

Thanks again! and looking forward to donating to publii!
